src/Context/BasicContext.php

```diff
@@ -54,6 +68,45 @@ class BasicContext implements Context
      */
     protected $datetimeFormat = 'Y-m-d H:i:s';
 
+    /**
+     * @var FixtureContext
+     */
+    protected $fixtureContext = null;
+
+    /**
+     * @BeforeScenario
+     */
+    public function gatherContexts(BeforeScenarioScope $scope)
+    {
+        /** @var InitializedContextEnvironment $environment */
+        $environment = $scope->getEnvironment();
+
+        // Find the FixtureContext defined in behat.yml
+        $subClasses = $this->getSubclassesOf(FixtureContext::class);
+        foreach ($subClasses as $class) {
+            if (!$environment->hasContextClass($class)) {
+                continue;
+            }
+            $this->fixtureContext = $environment->getContext($class);
+            break;
+        }
+        // Fallback to base FixtureClass
+        if (!$this->fixtureContext && $environment->hasContextClass(FixtureContext::class)) {
+            $this->fixtureContext = $environment->getContext(FixtureContext::class);
+        }
+    }
+
+    private function getSubclassesOf($parent)
```

Comment:
Clarify what this method is mean to do.
```suggestion
    private function getSubclassesOf($parent): array
```

Reply:
Updated